bosh-release/jobs/autoscaler_servicebroker/spec

```diff
@@ -24,8 +24,10 @@ properties:
     description: autoscaler_server password
   autoscaler_servicebroker.auth.username:
     description: autoscaler_server username
-  autoscaler.autoscaler_servicebroker.host:
+  autoscaler_servicebroker.host:
     description: the host for the autoscaler_servicebroker
+  autoscaler_servicebroker.name:
+    description: the autoscaler service name
```

Comment:
The new properties could probably use default values, so anybody who already have this (which _should_ be nobody) wouldn't need to change anything?